opsdroid/connector/github/__init__.py

```diff
@@ -18,12 +18,12 @@ class ConnectorGitHub(Connector):
     def __init__(self, config, opsdroid=None):
         """Create the connector."""
         super().__init__(config, opsdroid=opsdroid)
-        logging.debug("Loaded GitHub connector")
+        logging.debug("Loaded GitHub connector.")
         try:
             self.github_token = config["token"]
         except KeyError:
             _LOGGER.error(
-                _("Missing auth token!" "You must set 'token' in your config")
+                _("Missing auth token!" "You must set 'token' in your config.")
```

Comment:
This should be a single string like such:
```suggestion
                _("Missing auth token! You must set 'token' in your config.")
```

Again this is legacy logging and seems like if we are touching it we should correct it 😄